src/tmplits/tmplits-table/README.md

```diff
@@ -0,0 +1,95 @@
+# TmplitTable
+
+This is a container tmplit for TmplitTableRows. It can be given a title as the first context.
+
+<img src="example.png" width="400">
+
+# Usage
+
+```
+{{#tmplit 'Table' 'Inputs'}}
+        {{#tmplit 'TableRow'}}
+            <span>Channel</span>
+            <span>Description</span>
+            <span>Status</span>
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 0</span>
+            <span>This is an example of a digital input</span>
+            {{tmplit 'Led' data-var-name='tmplitTest:tmplit.SetButton'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 1</span>
+            <span>This is an example of a analog input</span>
+            {{tmplit 'Numeric' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 2</span>
+            <span>Description</span>
+            {{tmplit 'Numeric' data-var-name='tmplitTest:tmplit.NumericOutput'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 3</span>
+            <span>Description</span>
+            {{tmplit 'Led' data-var-name='tmplitTest:tmplit.SetButton'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 4</span>
+            <span>Description</span>
+            {{tmplit 'Led' data-var-name='tmplitTest:tmplit.SetButton'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 5</span>
+            <span>Description</span>
+            {{tmplit 'Led' data-var-name='tmplitTest:tmplit.SetButton'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 6</span>
+            <span>Description</span>
+            {{tmplit 'Led' data-var-name='tmplitTest:tmplit.SetButton'}}
+        {{/tmplit}}
+        {{#tmplit 'TableRow'}}
+            <span>Ch 7</span>
+            <span>Description</span>
+            {{tmplit 'Led' data-var-name='tmplitTest:tmplit.SetButton'}}
+        {{/tmplit}}
+    {{/tmplit}}
+```
+
+# CSS
+
+Below are the css vars used for styling. These are read into the css properties and are empty by default. If they are null the html uses static default values. See the example on how to populate these variable in an application:
+
+### Table Header
+| Property | Comment |
+| -------- | ------- |
+| tmplit-table-header-font-size ||
```

Comment:
Suggesting to follow [Material's lead ](https://github.com/material-components/material-web/blob/main/docs/components/button.md)when documenting design tokens, by including the full name and default value.

By full name, I mean: `--tmplit-table-header-font-size`

Reply:
The default or secondary value is especially useful to know when it is a material design token. Knowing this up-front would be helpful when designing.

And I suggest the full name, because it is more clear, and you can copy/paste it out of the docs.